repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -120,7 +120,7 @@ def _get_succinct_roles(api_url: str) -> SuccinctRoles:
 
 @admin.command()  # type: ignore
 @click.option(
-    "--api-url",
+    "--api-server",
     required=True,
```

Comment:
Why `--api-server` is required if the user can setup `settings.SERVER` to be used?

Reply:
Will love to hear your thoughts here.